Tasks/AzureFileCopyV4/AzureFileCopyRemoteJob.ps1

```diff
@@ -73,7 +74,11 @@ $AzureFileCopyRemoteJob = {
 
                 $azCopyFolderEnvPath = Join-Path -Path $azCopyFolderPath -ChildPath "AzCopy"
 
+                #setting path at machine level so that when user again do copy on VM, there is no need to download the azcopy.exe again
                 [Environment]::SetEnvironmentVariable("Path", $azCopyFolderEnvPath + ';' + $env:Path, [System.EnvironmentVariableTarget]::Machine)
+
+                #setting $env:Path to include azcopy.exe path as the above command used to set path at machine level takes some time to update the path environment variable
```

Comment:
no don't say it takes some time. What is happening here is that once the task process initiates, the $env:path variable is set. The path variable here is the user path variable. So after you set the path variable at machine level, the user path variable remains the same. Please update the comment to reflect this idea accurately.